src/main/java/org/jenkins/plugins/lockableresources/queue/LockRunListener.java

```diff
@@ -26,7 +26,7 @@
 import org.jenkins.plugins.lockableresources.LockableResourcesManager;
 import org.jenkins.plugins.lockableresources.actions.ResourceVariableNameAction;
 
-@Extension
+@Extension(optional = true)
```

Comment:
Please use `@OptionalExtension` instead, specifying the plugin. Using `Extension.optional` is less expressive, more error-prone (you _must_ throw a `LinkageError` while the extension is _instantiated_, not just _called_, in order to be effective), and at best it still logs a one-line warning which is unwanted (it does suppress the stack trace).